src/index.js

```diff
@@ -1 +1,21 @@
 import './style.css';
+import newGame from './modules/newGame.js';
+import postUserScore from './modules/formSubmit.js';
+import renderUserScores from './modules/renderScore.js';
+
+const formSubmitBtn = document.querySelector('#form-submit-btn');
+const refreshButton = document.getElementById('refresh-btn');
+
+window.addEventListener('load', () => {
+  newGame();
+});
```

Comment:
- [ ] [Optional] It would be better professionally if you render the scores on page load instead of making a new game.

Reply:
You are right, I realize I actually just need to call the new Game once to access the id. Thanks for this review. It's very helpful